master/internal/config/config.go

```diff
@@ -452,7 +462,8 @@ type SecurityConfig struct {
 
 // SSHConfig is the configuration setting for SSH.
 type SSHConfig struct {
-	RsaKeySize int `json:"rsa_key_size"`
+	RsaKeySize   int    `json:"rsa_key_size"`
+	CryptoSystem string `json:"crypto_system"`
```

Comment:
`crypto_system` seems like an unexpected phrase for what it means.  How about `key_type`?

For example, `man ssh-keygen says`:

>  The  type  of key to be generated is specified with the -t option.  If invoked without any arguments, ssh-keygen will generate an Ed25519 key.